kolibri/auth/api.py

```diff
@@ -72,7 +72,7 @@ class FacilityUserViewSet(viewsets.ModelViewSet):
 
     def create(self, request, *args, **kwargs):
         try:
-            return super(viewsets.ModelViewSet, self).create(request, *args, **kwargs)
+            return super(FacilityUserViewSet, self).create(request, *args, **kwargs)
```

Comment:
Ah yeah, I see now. You're calling the super method of this class' grandparent, not the parent.